Eatery/Controllers/Eateries/Campus/CampusEateryMealTableViewController.swift

```diff
@@ -65,7 +64,37 @@ class CampusEateryMealTableViewController: UITableViewController {
             return item.value.count
         } else {
             // display the menu items
-            return menu.data.count
+            let stations = sortedMenu!.map { $0.0 }
+            return menu.data[stations[section]]!.count
+        }
+    }
+    
+    override func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        guard let menu = menu, menu.data.count > section, tableView.numberOfSections > 1 else {
+            return nil
+        }
+        
+        let stationTitles = sortedMenu!.map { $0.0 }
+        
+        let headerView = UIView()
+        let sectionLabel = UILabel()
+        sectionLabel.text = stationTitles[section]
+        sectionLabel.font = .systemFont(ofSize: 18, weight: .medium)
+        headerView.addSubview(sectionLabel)
+        sectionLabel.snp.makeConstraints { make in
+            make.centerY.equalToSuperview()
+            make.leading.equalToSuperview().offset(10)
+            make.trailing.equalToSuperview().inset(10)
+        }
+
+        return headerView
```

Comment:
can we make a custom `UITableViewHeaderFooterView` and also use `tableView.dequeueReusableHeaderFooterView` in here so we are able to reuse it?